.github/workflows/dbdocs.yml

```diff
@@ -0,0 +1,61 @@
+name: build dbdocs
+
+# TODO: when done testing, change to "on push to main"
+on:
+  push
+
+env:
+  NODE_VERSION: 10
+  DB_NAME: cdhweb
+  DB_USER: cdhweb
+  DB_PASSWORD: cdhweb
+
+jobs:
+  python-unit:
+    services:
+      postgres:
+        image: postgres:12
+        env:
+          POSTGRES_DB: ${{ env.DB_NAME }}
+          POSTGRES_USER: ${{ env.DB_USER }}
+          POSTGRES_PASSWORD: ${{ env.DB_PASSWORD }}
+        ports:
+          - 5432:5432
+    name: Python unit tests
```

Comment:
Is all the database setup required? I guess the script won't run without a valid config even though it doesn't actually use the db?

Reply:
https://github.com/Princeton-CDH/cdh-web/runs/1559007185?check_suite_focus=true
Yeah! I tried removing it but it caused errors. But as I look again, I'm now thinking though that it's because I just didn't have the environment variable defined. I'll try dropping the postgres service and just running with:
```
- name: Set django database backend adapter
   run: echo "DJANGO_DB_BACKEND=postgresql" >> $GITHUB_ENV
```